cmp/grid/impl/ColumnHeader.js

```diff
@@ -66,11 +66,23 @@ export const columnHeader = hoistCmp.factory({
         ];
 
         let headerName = props.displayName;
-        if (impl.xhColumn && isFunction(impl.xhColumn.headerName)) {
+        if (isFunction(impl.xhColumn?.headerName)) {
             const {xhColumn, gridModel} = impl;
             headerName = xhColumn.headerName({column: xhColumn, gridModel});
         }
 
+        const showTooltipWhenElided = isNil(impl.xhColumn?.headerTooltip),
```

Comment:
Good catch thanks was just about to suggest

Reply:
Or rather, I did suggest - didn't realize I had submitted that comment immediately :)